pkg/reader/ccip.go

```diff
@@ -916,6 +929,34 @@ func (r *ccipChainReader) getFeeQuoterTokenPriceUSD(ctx context.Context, tokenAd
 	return cciptypes.NewBigInt(price), nil
 }
 
+// destChainConfig is used to parse the response from the feeQuoter contract's getDestChainConfig method.
+// See: https://github.com/smartcontractkit/chainlink/blob/23452266132228234312947660374fb393e96f1a/contracts/src/v0.8/ccip/FeeQuoter.sol#L1005
+//
+//nolint:lll // It's a URL.
+type destChainConfig struct {
+	DestDataAvailabilityOverheadGas   uint32 `json:"destDataAvailabilityOverheadGas"`
+	DestGasPerDataAvailabilityByte    uint16 `json:"destGasPerDataAvailabilityByte"`
+	DestDataAvailabilityMultiplierBps uint16 `json:"destDataAvailabilityMultiplierBps"`
+}
+
+// getDestChainConfig returns the destination chain's Fee Quoter's DestChainConfig
+func (r *ccipChainReader) getDestChainConfig(ctx context.Context) (destChainConfig, error) {
+	var dynamicConfig destChainConfig
+	err := r.getDestinationData(
```

Comment:
I think there's a few issues here. First, I don't think `GetDynamicConfig` is a FeeQuoter method. I think the correct method is `getDestChainConfig`, but this function takes `destChainSelector` as a param, so that would need to be supplied. Also, we can't get the dest chain's `FeeQuoter.DestChainConfig` from the FeeQuoter instance on the dest chain, [see this thread](https://chainlink-core.slack.com/archives/C02PBKHCP28/p1729635837527769). So we'll need to get `DestChainConfig` from the FeeQuoter on a different chain, but maybe there's a better way. I think we'd also need to ensure that the encoded `DestChainConfig` we get from FeeQuoter can be parsed into the elided/simplified `destChainConfig` struct we've defined above.

Reply:
As discussed offline, we will fetch all the possible configs for a dest chain and take the median